### DISCUSSION TITLE
v6 and custom react renderer (RouterContext?)

### DISCUSSION THREAD

**Discussion by giraud** [remix-run/react-router#10161] [v5 to v6 Migration]
Hi,

I am trying to migrate to v6 and have a problem with the router context. 

I am using a custom react renderer, and context are not propagated between reconcilers. with v5 I had a bridge component that was just importing the router context with useRouterContext and reexporting it (with a component as a child of the custom renderer).

I need to use the Link component in the custom renderer but now I have no idea how to do it in v6. I tried to read the different workaround in the issues but I can’t find a working solution. 
I can recreate a RouterProvider to be used the custom renderer, but when the URL is updated, it has no impact on the parent provider: url is updated but navigation does not happen, I need to use the same instance.

Any hint ?
thanks

**Comment by davidrichard23** [comment]
I'm running into the same issue. Did you ever find a solution?

**Comment by giraud** [comment]
no solution so far
